cmake/opendds_testing.cmake

```
@@ -27,6 +27,13 @@ function(opendds_add_test)
   else()
     set(env_var_name LD_LIBRARY_PATH)
   endif()
+  if(DEFINED _OPENDDS_GOOGLE_TEST_DIR)
+    _opendds_path_list(lib_dir_list APPEND "${_OPENDDS_GOOGLE_TEST_DIR}")
+    if(CMAKE_CONFIGURATION_TYPES)
+      _opendds_path_list(lib_dir_list APPEND
+        "${_OPENDDS_GOOGLE_TEST_DIR}$<$<BOOL:$<CONFIG>>:/$<CONFIG>>")
+    endif()
+  endif()
   _opendds_path_list(lib_dir_list "$ENV{${env_var_name}}" "${TAO_LIB_DIR}")
```

Somehow the `if(DEFINED _OPENDDS_GOOGLE_TEST_DIR)...` and the lone ` _opendds_path_list` got swapped between testing and when I submitted the PR. This should fix the char_literals test.

```suggestion
  _opendds_path_list(lib_dir_list "$ENV{${env_var_name}}" "${TAO_LIB_DIR}")
  if(DEFINED _OPENDDS_GOOGLE_TEST_DIR)
    _opendds_path_list(lib_dir_list APPEND "${_OPENDDS_GOOGLE_TEST_DIR}")
    if(CMAKE_CONFIGURATION_TYPES)
      _opendds_path_list(lib_dir_list APPEND
        "${_OPENDDS_GOOGLE_TEST_DIR}$<$<BOOL:$<CONFIG>>:/$<CONFIG>>")
    endif()
  endif()
```